src/rust/ide/view/graph-editor/src/component/node/action_bar.rs

```diff
@@ -175,24 +178,29 @@ impl Model {
         let offset         = BUTTON_OFFSET;
         let hover_padding  = 1.0;
         let button_width   = self.icon_size().x;
-        let hover_width    = button_width * (button_count + hover_padding + offset + padding);
+        let hover_width    = button_width * (button_count + hover_padding + offset + padding)
+            + HOVER_PADDING_X;
         let hover_height   = button_width * 2.0;
         let hover_ara_size = Vector2::new(hover_width,hover_height);
         self.hover_area.size.set(hover_ara_size);
         let center_offset  = -size.x / 2.0 + hover_ara_size.x / 2.0;
-        let padding_offset = - 0.5 * hover_padding * button_width;
+        let padding_offset = - 0.5 * hover_padding * button_width - HOVER_PADDING_X / 2.0;
         self.hover_area.set_position_x(center_offset + padding_offset);
     }
 
     fn set_size(&self, size:Vector2) {
         self.size.set(size);
         self.icons.set_position_x(-size.x/2.0);
 
+        // Note: Disabled for https://github.com/enso-org/ide/issues/1397
+        //  Re-enable when they are needed again. Also increase the number in
+        // the call to `layout_hover_area_to_cover_buttons` belolw.
```

Comment:
The same comment as previously. This sohuld describe why it was disabled etc (See prev comment for details).